invenio.cfg

```diff
@@ -202,10 +202,17 @@ _openaire_aai_sandbox_app = KeycloakSettingsHelper(
     description="Open Science Services.",
     base_url="https://beta.aai.openaire.eu",
     realm="openaire",
-    scopes="openid profile email",
+    scopes="openid profile email eduperson_entitlement orcid",
```

Comment:
major: if oauthclient is merged, we can import the whole remote app and remove from line 200